elementary/monitor/api/tests/schema.py

```diff
@@ -27,10 +27,13 @@ class DbtTestResultSchema(BaseModel):
     failed_rows_count: Optional[int] = None
 
 
-class TestMetadataSchema(BaseModel):
+class RawTestResultSchema(BaseModel):
```

Comment:
I think Pydantic models are not meant to have methods like that.
I think their main purposes should be data classes, in the sense that they hold validated data.

Reply:
I am not sure what do you mean by methods like that?
Maybe the comment is on the wrong line?

Reply:
I moved the more logical methods outside of the schema, so now it contains only data validation 🙂